README.md

```diff
@@ -144,32 +151,38 @@ Play with trading simulators where you can engage with the market and practice y
  * [Coinmarketcap](https://coinmarketcap.com/) - CoinMarketCap provides information about all digital currencies that are traded in at least one public exchange.
  * [Coingecko](https://www.coingecko.com/en) - Like CMC, CoinGecko is a platform for tracking cryptocurrencies and blockchain assets.
  * [Coin360](https://coin360.com/) - Heatmap of cryptocurrency prices, market capitalizations, and volumes.
- 
+ * [Crypto Compare](https://www.cryptocompare.com/) - Like CMC, Crypto Compare is a platform for tracking cryptocurrencies and blockchain assets.
+ * [Crypto Hedgehog](https://cryptohedgehog.io/) - Crypto Hedgehog aggregates data from CoinMarketCap, Coin Gecko, Crypto Compare, Messari, Santiment, etc. Research includes market data, overviews, teams, partners, ICO information, feeds, and more.
+ * [Messari](https://messari.io/) - Messari provides reliable data and market intelligence for crypto investors and professionals.
```

Comment:
Messari seems to be more like a tool designed for market screening. Let's add their market screener (https://messari.io/screener) to "Market Screeners" section instead.